Access Engineering.js

```diff
@@ -295,23 +346,23 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "https://www.accessengineeringlibrary.com/content/tutorial/T0002_Open_Channel_Flow_Calculations_with_the_Manning_Equation",
+		"url": "https://www.accessengineeringlibrary.com/content/tutorial/T0004_Partially_Full_Pipe_Flow_Calculations_Using_Excel_Spreadsheets",
 		"items": [
 			{
 				"itemType": "journalArticle",
-				"title": "Open Channel Flow Calculations with the Manning Equation using Excel Spreadsheets",
+				"title": "Partially Full Pipe Flow Calculations Using Excel Spreadsheets",
 				"creators": [
 					{
-						"firstName": "Harlan",
-						"lastName": "H. Bengtson",
+						"firstName": "Harlan H.",
+						"lastName": "Bengtson",
 						"creatorType": "author"
 					}
 				],
-				"date": "2014-02-01",
-				"abstractNote": "This tutorial teaches the Manning equation and its use for uniform open channel flow calculations, including the hydraulic radius, Manning roughness coefficient, and normal depth. There are example problems and illustrations show how to use spreadsheets for the calculations.",
+				"date": "2014/02/01/",
```

Comment:
Would be good to change these to YYYY-MM-DD in a separate update.

Reply:
@dstillman Thank you. I will check and do the same in separate update.